### ISSUE TITLE
the loss curve during the pretraining phase

### ISSUE THREAD

**Issue by 20191864218** [haotian-liu/LLaVA#1263]
### Question

I replaced the vision encoder and LLM (Qwen7B-base) with LLaVA's original training data for pretraining. Here is the loss curve.

![image](https://github.com/haotian-liu/LLaVA/assets/56297762/3bff5f87-d660-4dd2-8d7b-5f1a49db19fd)
